src/screens/AddSiteFormScreen.js

```diff
@@ -260,12 +294,12 @@ class AddSiteFormScreen extends Component {
     };
 
     render() {
-        const {buttonStyle, headerTitle, largeTextInput, modalStyle, imageRowStyle} = styles;
-        const {latitudeText, longitudeText, siteTitleText, siteDescriptionText, siteDirectionsText, siteNearestTownText, accessibilityOption, facilitiesOption, priceOption} = this.props;
+        const {iAmHereButtonStyle, headerTitle, largeTextInput, modalStyle, imageRowStyle, labelStyle, formInputStyle, pickerStyle, cellServiceContainerStyle, cellProviderPickerStyle, cellStrengthPickerStyle} = styles;
+        const {latitudeText, longitudeText, siteTitleText, siteDescriptionText, siteDirectionsText, siteNearestTownText, accessibilityOption, facilitiesOption, priceOption, siteAlternateSitesText, cellProviderOption, cellStrengthOption} = this.props;
 
         return (
-            <View>
-                <ScrollView style={modalStyle}>
+            <KeyboardAwareScrollView>
```

Comment:
Let's see if `KeyboardAwareScrollView` works out. If it does, it would be great, as all we had to do to implement is is wrap the form input area in the `< KeyboardAwareScrollView>` tag...